app/assets/stylesheets/moj/_forms.scss

```diff
@@ -14,6 +14,11 @@ form {
       margin: 20px 0 !important;
     }
   }
+
+  .form-buttons * {
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2263/files#r171911547)